src/components/KInputSwitch/KInputSwitch.cy.ts

```diff
@@ -25,15 +25,4 @@ describe('KInputSwitch', () => {
       cy.wrap(Cypress.vueWrapper.emitted()).should('have.property', 'update:modelValue')
     })
   })
-
-  it('shows enabled icon if prop is set', () => {
```

Comment:
If there isn't one already, can you add a test for the disabled state (e.g. can't change the value)